views/pages/practitioner/declaration.njk

```diff
@@ -0,0 +1,354 @@
+{% extends "layouts/layout.njk" %}
+
+{% block content %}
+
+    {{ backButton('dashboard') }}
+
+    <p>{{ __('declaration.review_and_confirm') }}</p>
+
+    <details class="mt-4 border-none" open>
+        <summary>
+            <h2 class="mt-4 text-4xl inline">{{ __('dashboard.basic_info') }}</h2>
```

Comment:
Is it possible to make it more obvious that the section titles are links? Maybe using the `.cursor-pointer` class (pointy hand.

Reply:
done